c/driver/postgresql/postgresql_test.cc

```diff
@@ -63,6 +63,24 @@ class PostgresQuirks : public adbc_validation::DriverQuirks {
     return status;
   }
 
+  AdbcStatusCode DropView(struct AdbcConnection* connection, const std::string& name,
```

Comment:
Instead of a new method I think could also parametrize `DropTable` to accept a constant table type argument. That would be more consistent with other method names that refer to everything as `Table`